firebase/firestore.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /alerts-subscriptions/{email} {
+      allow write;
+    }
+  }
```

Comment:
FWIW- These rules allow anybody to write to /alerts-subscriptions/{email}, but they can't read, which is suitable for our use case where we just want to let people write (and delete) their subscriptions.